opencti-platform/opencti-graphql/src/initialization.js

```diff
@@ -340,6 +340,7 @@ const platformInit = async (withMarkings = true) => {
       const reason = '[OPENCTI] Platform cant get the lock for initialization (can be due to other instance currently migrating/initializing)';
       throw LockTimeoutError({ participantIds: [PLATFORM_LOCK_ID] }, reason);
     } else {
+      logApp.error('[OPENCTI] Platform initialization fail', e);
```

Comment:
Useful for testing but not in runtime that will duplicate the log.
For information e is also usually wrapped into error filed